Quoteau/Podfile

```diff
@@ -0,0 +1,12 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'Quoteau' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for Quoteau
+
+pod 'SnapKit', '~> 4.0'
```

Comment:
```suggestion
  pod 'SnapKit', '~> 4.0'
```

Reply:
By  the way, we should probably use swift 5. Then we should also use snapkit > 5.0.0